README.md

```diff
@@ -1,8 +1,8 @@
-# py.test plugin for the sublime text editor
+# py.test integration for Sublime Text 3
 
-The plugin basically runs your tests, and annotates your files using the tracebacks.
+This plugin runs your tests and on failure annotates your test files using the tracebacks.
```

Comment:
LGTM except it does annotate all files not just test files.

Reply:
I have only noticed it annotating my test files. Anyhow corrected the sentence.